pipeline/extract_generation.py

```diff
@@ -48,7 +48,7 @@ def construct_default_params(self) -> Dict[str, str]:
         Get a time range. Currently fetches data from 12 hours ago to present.
         """
         twelve_hours_ago = datetime.now(timezone.utc) - timedelta(hours=12)
-        now = datetime.now(timezone.utc) - timedelta(hours=1)
+        now = datetime.now(timezone.utc)
```

Comment:
Glad we removed this small transform from extract. I'm very keen on the separation of concerns. All data manipulation choices should occur in transform so we always know where to look.